addons/mail/static/src/models/message/message.js

```diff
@@ -665,6 +666,12 @@ function factory(dependencies) {
         is_notification: attr({
             default: false,
         }),
+        /**
+         * Mark the message as pending: not sent to the server yet.
+         */
+        isPending: attr({
+            default: false,
+        }),
```

Comment:
`isPendingSend`
Default value should be undefined (no need to specify it) because it's not applicable for most messages.
`false` should be set when creating a new temporary message with the goal to send it.